config/purposes/scrna_core_cell_extraction.yml

```diff
@@ -110,6 +107,8 @@ LRC TR Bank Seq:
       id: tube_rack_concentrations_ngul
     - name: Download Racked Tube Concentrations (nM) CSV
       id: tube_rack_concentrations_nm
+    - name: 'Download Hamilton LRC PBMC Bank plate to LRC Bank Seq and Spare tubes CSV'
+      id: hamilton_lrc_pbmc_bank_to_lrc_bank_seq_and_spare
```

Comment:
Any reason for changing the order? I have a feeling the order of these was discussed and agreed (think it controls the order in the UI).

Reply:
By order do you mean order of files in the UI? If so I am not sure what order they should be in.  But this download has moved from the parent plate to the tube racks as suggested by @andrewsparkes as the UI ends up on the tube racks page after performing the tube rack split from the PBMC bank plate.

Reply:
For this one I'd put these tube rack concentration file downloads in, as we have similar ones for plates.
A better question is would they / could they even do QC on a tube rack of tubes, so do we even need those file download links?